assets/search/components/TopicForm.jsx

```diff
@@ -39,14 +39,17 @@ const TopicForm = ({original, topic, save, onChange, globalTopicsEnabled, onSubs
                     readOnly={readOnly}
                 />
             )}
-            {!globalTopicsEnabled ? null : (
+            {!(globalTopicsEnabled && original.user) ? null : (
```

Comment:
This will remove the `Share with my Company` checkbox when creating a new `Topic`.

Reply:
When we create `topic` from UI, It is visible in `My Topics`. Then I have to click on `Share with my Company` option available to move it to `company topics`.
But when I create a topic from `API` without `User`, it get's a label `shared externally`.

Reply:
The user should be able to create the Topic as a Company Topic when they initially create.
Therefor this checkbox should be available when creating a topic in the UI, not just updating the topic

Reply:
Ohh, okay. that means it is totally unnecessary.

Reply:
No it is necessary to remove the checkbox, if the topic is an existing topic